src/Application/FavoriteSchools/FavoriteSchools.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import SchoolCard from '../SchoolCard/SchoolCard';
+
+const FavoriteSchools = ({ favoriteSchools }) => {
+
+  // const [favorites, setFavorites] = 
+  const returnFavoriteSchoolCards = () => {
+    const favoriteSchoolCards = favoriteSchools.map((list) => {
+      return (
+        <SchoolCard 
+          key={Math.random()} 
```

Comment:
If you have two arguments in your map function, like below, then you can use the index as the key on the card.

... favoriteSchools.map((list, index) => ...